docs/src/modules/utils/useRouterExtra.ts

```diff
@@ -2,8 +2,9 @@ import { useRouter } from 'next/router';
 
 export default function useRouterExtra() {
   const router = useRouter();
-  const asPathWithoutLang = router.asPath.replace(/^\/[a-zA-Z]{2}\//, '/');
-  let product = asPathWithoutLang.replace(/^\/([^/]+)\/.*/, '$1');
+
+  const asPathWithoutLang = router.asPath.replace(/^\/+[a-zA-Z]{2}\//, '/');
+  let product = asPathWithoutLang.replace(/^\/+([^/]+)\/.*/, '$1');
   if (asPathWithoutLang.startsWith('/x')) {
```

Comment:
In case we have URLs that also start with /x?

```suggestion
  if (asPathWithoutLang.startsWith('/x/')) {
```